src/libsystemd-network/sd-dhcp6-lease.c

```diff
@@ -809,9 +894,11 @@ static int dhcp6_lease_parse_message(
 }
 
 static sd_dhcp6_lease *dhcp6_lease_free(sd_dhcp6_lease *lease) {
+
```

Comment:
Please do not insert unnecessary empty line.

Reply:
Removed the extra new line